libPhoneNumber/NBPhoneNumberUtil.m

```diff
@@ -2622,6 +2622,61 @@ - (BOOL)isPossibleNumber:(NBPhoneNumber*)number
 }
 
 
+- (NBEValidationResult)testNumberLength:(NSString *)number metadata:(NBPhoneMetaData *)metadata {
```

Comment:
it's the name of the method in the original lib, and  it can just be harder to compare if someone tries to compare them. But I can change it to something else, I guess.